code/modules/fishing/fish/fish_types.dm

```diff
@@ -181,6 +181,21 @@
 	fillet_type = /obj/item/food/fishmeat/armorfish
 	fish_ai_type = FISH_AI_SLOW
 
+//Chasm fish
+/obj/item/fish/chrab
```

Comment:
```suggestion
/obj/item/fish/chasm_crab
```
a better path is easier to actually know what this is without going to the defintion